src/components/mote/components/innkalling/InnkallingDialogmotePanel.tsx

```diff
@@ -37,6 +40,16 @@ export const InnkallingDialogmotePanel = ({
   const { hasActiveOppfolgingstilfelle } = useOppfolgingstilfellePersonQuery();
   const { isKandidat } = useDialogmotekandidat();
 
+  const DialogmoteunntakSkjemaLenke = () => {
+    return (
+      <Link to={dialogmoteUnntakRoutePath}>
+        <Button as="a" variant="secondary" href={dialogmoteUnntakRoutePath}>
+          {texts.settUnntakButton}
+        </Button>
+      </Link>
```

Comment:
Synes dette er litt klønete. Må vi bruke `Link`? 🤔

Reply:
Snublet over https://aksel.nav.no/grunnleggende/kode/overridablecomponent og det ser ut til at man kan droppe `<Link>` rundt og heller gjøre `<Button as={Link} to={dialogmoteUnntakRoutePath} ...>`. :bulb:

Reply:
Diigg! 😍